gubernator/templates/build.html

```diff
@@ -9,7 +9,7 @@
 <script>window.onload = fix_timestamps;</script>
 <div id="header" class="container">
 <img src="/static/logo.svg">
-<h1>{% if pr %}<a href="/pr/{{pr}}">PR #{{pr}}</a> {% endif %}<a href="/builds{{job_dir}}">{{job}}</a> #{{build}}</h1>
+<h1>{% if pr %}<a href="/pr/{{pr}}">PR #{{pr}}</a> {% endif %} <a href="/builds{{job_dir|dirname}}?local={{local}}">{{job}}</a> #{{build}}</h1>
```

Comment:
Are you adding local in all these places just to pass the query parameter along (so it's not lost when navigating)? If so, I think there should be a way with Angular to install a navigation handler that preserves whitelist parameters.